plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java

```diff
@@ -107,11 +105,17 @@
     private final JsonCodec<List<ExtraColumnMetadata>> extraColumnMetadataCodec;
     private final Supplier<CqlSession> session;
     private final Duration noHostAvailableRetryTimeout;
+    private final CassandraTypeManager cassandraTypeManager;
 
-    public CassandraSession(JsonCodec<List<ExtraColumnMetadata>> extraColumnMetadataCodec, Supplier<CqlSession> sessionSupplier, Duration noHostAvailableRetryTimeout)
+    public CassandraSession(
+            JsonCodec<List<ExtraColumnMetadata>> extraColumnMetadataCodec,
+            Supplier<CqlSession> sessionSupplier,
+            Duration noHostAvailableRetryTimeout,
+            CassandraTypeManager cassandraTypeManager)
```

Comment:
Let's reorder `CassandraTypeManager` to the 1st argument (including `private final...` & `this.cassandraTypeManager...`). Sorry, I know it's comes from my rough example.